mock/etc/mock/site-defaults.cfg

```diff
@@ -128,8 +128,8 @@
 
 # when 'use_bootstrap_container' is True, these commands are used to build
 # the minimal chroot for the respective package manager
-# config_opts['yum_install_command'] = 'install yum yum-utils shadow-utils distribution-gpg-keys'
-# config_opts['dnf_install_command'] = 'install dnf dnf-plugins-core shadow-utils distribution-gpg-keys'
+# config_opts['yum_install_command'] = 'install yum yum-utils distribution-gpg-keys'
+# config_opts['dnf_install_command'] = 'install dnf dnf-plugins-core distribution-gpg-keys'
```

Comment:
But if we **use** bootstrap, then we need it. Right?

Reply:
I don't think so, that's why I submitted this PR...  but I may be wrong.  Any idea why the user would be needed in bootstrap chroot?